src/operators/Tessellate/QvisTessellateWindow.C

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+#include "QvisTessellateWindow.h"
+
+#include <TessellateAttributes.h>
+
+#include <QCheckBox>
+#include <QLabel>
+#include <QLayout>
+#include <QLineEdit>
+#include <QSpinBox>
+#include <QButtonGroup>
+#include <QRadioButton>
+#include <QvisVariableButton.h>
+
+
+// ****************************************************************************
+// Method: QvisTessellateWindow::QvisTessellateWindow
+//
+// Purpose: 
+//   Constructor
+//
+// Note:       Autogenerated by xml2window.
+//
+// Programmer: xml2window
+// Creation:   omitted
+//
+// Modifications:
+//   
+// ****************************************************************************
+
+QvisTessellateWindow::QvisTessellateWindow(const int type,
+                         TessellateAttributes *subj,
+                         const QString &caption,
+                         const QString &shortName,
+                         QvisNotepadArea *notepad)
+    : QvisOperatorWindow(type,subj, caption, shortName, notepad)
+{
+    atts = subj;
+}
+
+
+// ****************************************************************************
+// Method: QvisTessellateWindow::~QvisTessellateWindow
+//
+// Purpose: 
+//   Destructor
+//
+// Note:       Autogenerated by xml2window.
+//
+// Programmer: xml2window
+// Creation:   omitted
+//
+// Modifications:
+//   
+// ****************************************************************************
+
+QvisTessellateWindow::~QvisTessellateWindow()
+{
+}
+
+
+// ****************************************************************************
+// Method: QvisTessellateWindow::CreateWindowContents
+//
+// Purpose: 
+//   Creates the widgets for the window.
+//
+// Note:       Autogenerated by xml2window.
+//
+// Programmer: xml2window
+// Creation:   omitted
+//
+// Modifications:
+//   
+// ****************************************************************************
+
+void
+QvisTessellateWindow::CreateWindowContents()
+{
+    QGridLayout *mainLayout = new QGridLayout(0);
+    topLayout->addLayout(mainLayout);
+
+    chordErrorLabel = new QLabel(tr("Chord error"), central);
+    mainLayout->addWidget(chordErrorLabel,0,0);
+    chordError = new QLineEdit(central);
+    connect(chordError, SIGNAL(returnPressed()),
```

Comment:
Will this signal on tab as well? an [editingFinished](https://doc.qt.io/qt-5/qlineedit.html#editingFinished) signal may be better.

Reply:
Completed.